src/modules/rdc.sh

```diff
@@ -0,0 +1,15 @@
+AMBER_RDC_MD=()
+for d in "${AMBER_RDC_CD[@]}"
+do
+    if ! command -v $d > /dev/null 2>&1; then
+        AMBER_RDC_MD+=($d)
+    fi
+done
+
+if (( ${#AMBER_RDC_MD[@]} != 0 )); then
+    >&2 echo This program requires for these commands: \( $AMBER_RDC_MD \) to be present in \$PATH.
```

Comment:
The "for" in here breaks the english grammar. "This program requires these commands:"  would be more correct.